Functions.Templates/Templates-v2/DurableFunctions-Python/template.json

```diff
@@ -0,0 +1,176 @@
+{
+    "author": "Varad Meru (@vrdmr)",
+    "name": "Blob trigger",
+    "description": "$BlobTrigger_description",
+    "programmingModel": "v2",
+    "language": "python",
+    "jobs": [
+        {
+            "name": "Create New Project with Durable Function",
+            "type": "CreateNewApp",
+            "inputs": [
+                {
+                    "assignTo": "$(APP_FILENAME)",
+                    "paramId": "app-fileName",
+                    "defaultValue": "function_app.py",
+                    "required": true,
+                    "condition" : {
+                        "name": "ClientId",
+                        "values": [ "VSCode"],
+                        "operator": "IN"
+                    }
+                },
+                {
+                    "assignTo": "$(ORCHESTRATOR_FUNCTION_NAME)",
+                    "paramId": "orchestrator-functionName",
```

Comment:
Please declare this parameter in `/Bindings/UserPrompts.json`.
You can follow this example: https://github.com/Azure/azure-functions-templates/blob/ceca1157044975bad1706ea17fb070b1bbb868bb/Functions.Templates/Bindings/userPrompts.json#L94-L99

Reply:
Thanks! I changed this parameter to an existing one `trigger-functionName`. Cause it seems that I could use the same one for different folders. I think this could be more convenient.